app/views/ChoosingMethodView.scala.html

```diff
@@ -41,8 +41,6 @@
             call = Call("GET", config.overviewForMethodsUrl)
         )}
 
-@legendH2 = {@subheading(messages("valuationMethod.h2"))}
```

Comment:
This was removed which is OK as it is unused, however, it is still present in the messages files. Please can they be also removed in those files? Thanks